libs/sdk-react-native/ios/RNBreezSDK.swift

```diff
@@ -111,6 +129,26 @@ class RNBreezSDK: RCTEventEmitter {
             reject(RNBreezSDK.TAG, "Invalid config", nil)
         }
     }
+
+    @objc(signMessage:resolver:rejecter:)
+    func signMessage(_ request: SignMessageRequest, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> SignMessageResponse {
+        do {
+            let response = try BreezSDK.sign_message(message)
+            resolve(response)
+        } catch let error {
+            reject(RNBreezSDK.TAG, "Error calling signMessage", error)
+        }
+    }
+
+    @objc(checkMessage:pubkey:signature:resolver:rejecter:)
+    func checkMessage(_ request: CheckMessageRequest, resolver resolve: @escaping RCTPromiseResolveBlock, rejecter reject: @escaping RCTPromiseRejectBlock) -> CheckMessageResponse {
+        do {
+            let response = try BreezSDK.check_message(request)
+            resolve(response)
+        } catch let error {
+            reject(RNBreezSDK.TAG, "Error calling checkMessage", error)
+        }
```

Comment:
This function should convert the reqData into a CheckMessageRequest struct, then return a dictionary mapping the CheckMessageResponse

```
        if let checkMessageRequest = BreezSDKMapper.asCheckMessageRequest(reqData: reqData) {
            do {
                let checkMessageResponse = try getBreezServices().checkMessage(reqData: checkMessageRequest)                
                resolve(BreezSDKMapper.dictionaryOf(checkMessageResponse: checkMessageResponse))
            } catch let err {
                rejectErr(err: err, reject: reject)      
            }
        } else {            
            rejectErr(err: SdkError.Generic(message:"Invalid reqData"), reject: reject)
        }
```